### PR TITLE
Project Setup - Configure Webpack

### PR DESCRIPTION
**Pull request by reshma-qburst**
Initial Project set up - npm build scripts & webpack configuration

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+	selector: 'my-app',
```
Comment (nishintn) [inline src/app/app.component.ts]: Please be professional. How can we this code to a client?

File: src/app/app.component.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+	template: '<h1>Hello {{name}} !!!</h1>'
+})
+
+export class AppComponent {
+	name = "John";
```
Comment (nishintn) [inline src/app/app.component.ts]: John?

File: src/app/app.component.ts (hunk 1 above)
Comment (reshma-qburst) [inline src/app/app.component.ts]: Have amended the selector variable name.

File: src/app/app.component.ts (hunk 2 above)
Comment (reshma-qburst) [inline src/app/app.component.ts]: This was a change in the very first commit. Already removed in later commits.

**Pull request merged**